pkg/commands/loading_remotes.go

```diff
@@ -30,13 +32,40 @@ func (c *GitCommand) GetRemotes() ([]*models.Remote, error) {
 		re := regexp.MustCompile(fmt.Sprintf(`%s\/([\S]+)`, remoteName))
 		matches := re.FindAllStringSubmatch(remoteBranchesStr, -1)
 		branches := make([]*models.RemoteBranch, len(matches))
+
 		for j, match := range matches {
-			branches[j] = &models.RemoteBranch{
+			branch := &models.RemoteBranch{
```

Comment:
could we instead make use of the sort arg in the git branch command? See https://git-scm.com/docs/git-branch#Documentation/git-branch.txt---sortltkeygt